Specs/Number/French/NumberRangeModel

```diff
@@ -0,0 +1,1359 @@
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "pas moins de trente",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[30,)"
+        },
+        "Start": 7,
+        "End": 26
+      }
+    ]
+  },
+  {
+    "Input": "Il y en a plus de 500 dans ces produits.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "plus de 500",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[500,)"
+        },
+        "Start": 10,
+        "End": 21
+      }
+    ]
+  },
+  {
+    "Input": "Il y en a environ 500 ou plus dans ces produits.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "environ 500 ou plus",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "[500,)"
+        },
+        "Start": 10,
+        "End": 29
+      }
+    ]
+  },
+  {
+    "Input": "Plus de la moitié des gens sont venus ici.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "Plus de la moitié",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(0.5,)"
+        },
+        "Start": 0,
+        "End": 17
+      }
+    ]
+  },
+  {
+    "Input": "Trouvez les nombres premiers qui sont plus petits ou égaux à 100",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "plus petits ou égaux à 100",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,100]"
+        },
+        "Start": 38,
+        "End": 64
+      }
+    ]
+  },
+  {
+    "Input": "Trouvez les nombres premiers qui sont inférieurs ou égaux à 100",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "inférieurs ou égaux à 100",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,100]"
+        },
+        "Start": 38,
+        "End": 63
+      }
+    ]
+  },
+  {
+    "Input": "Il y en a 500 ou moins dans ces produits.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "500 ou moins",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,500]"
+        },
+        "Start": 10,
+        "End": 22
+      }
+    ]
+  },
+  {
+    "Input": "Trouvez les nombres premiers qui sont < = 100",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "< = 100",
+        "TypeName": "numberrange",
+        "Resolution": {
+          "value": "(,100]"
+        },
+        "Start": 38,
+        "End": 45
+      }
+    ]
+  },
+  {
+    "Input": "Sa taille est en dessuous de 170.",
+    "NotSupported": "dotnet, javascript, python, java",
+    "Results": [
+      {
+        "Text": "en dessuous de 170",
```

Comment:
Why should the "en" be included?

Reply:
The whole phrase "en dessous de" = under